Cargo.toml

```diff
@@ -119,8 +119,8 @@ toml = "0.8"
 toml_edit = { version = "0.22", features = ["serde"] }
 tower-http = { version = "0.4", features = ["fs"] }
 tracing = "0.1"
-tracing-log = "0.2"
 tracing-core = "0.1"
+tracing-log = "0.2"
```

Comment:
why?

Reply:
It just got moved one line below D:

Reply:
I've used this
```
cargo xtask set-cairo-version --rev 290f51f554c978180ca9d91177423305959cbed5
```

Reply:
It sorts the deps from what I see
https://github.com/software-mansion/scarb/blob/110826a7abf9b61d6dcee7da38a1538fb1298470/xtask/src/set_cairo_version.rs#L79

Reply:
Cool, we must have missed it then!